lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,20 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
```

Comment:
@bizla that isn't equivalent to what I have. In my code if `STANDARD_ERROR_CODE_MAPPING[decline_code]` doesn't have a mapping, the code falls back to `STANDARD_ERROR_CODE_MAPPING[code]`. Your version doesn't do that.